[IMP] Contacts: memberships and partners tabs (#9779)
odoo/documentation: content/applications/essentials/contacts.rst

Review` configurations. These are **only** present when the *Enterprise Resellers* module is
installed.

> To publish a list of all active partners on the website, a new webpage must be created first.
> Navigate to :menuselection:`Website app --> Site --> Pages` and click :guilabel:`New`. Hover over
> the :guilabel:`Blank Page` template, and click the :guilabel:`Use this template` button to create a
> new page.

Reviewer: ```suggestion
To publish a list of all active partners on the website, a new webpage must be created first.

To create a new webpage, navigate to :menuselection:`Website app --> Site --> Pages`, and click :guilabel:`New`. Hover over
the :guilabel:`Blank Page` template, and click the :guilabel:`Use this template` button to create a
new page.
```
SUGGESTION: broke up the paragraph, added comma for enhanced sentence flow/pacing

This is a totally viable way to create a new page. However, I think the same can be accomplished by going to Website app --> Clicking +New in the upper-right corner --> Selecting 'Page' from the presented options --> and selecting the 'Blank Page' template as you've instructed above.

Just something to consider adding during your revisions.

> In the :guilabel:`New Page` pop-up window, enter `Partners` in the :guilabel:`Page Title` field.

Reviewer: ```suggestion
In the :guilabel:`New Page` pop-up window that appears, enter an appropriate title, like `Partners`, in the :guilabel:`Page Title` field.
```
just some added specificity.

also, do you want to consider mentioning anything about the 'Add to menu' toggle switch that is present in this pop-up window? It could be beneficial to briefly mention this to the reader, as well. Just a suggestion...

> Then click :guilabel:`Create`. In the :guilabel:`Edit` toolbar on the far right of the page, click

Reviewer 2: ```suggestion
Then, click :guilabel:`Create`. In the :guilabel:`Edit` toolbar on the far-right of the page, click
```

> :guilabel:`Save`.

Reviewer: Do you think it would be beneficial to briefly allude to how the user can make any necessary edits/adjustments to the page, if desired, from this toolbar? Or, do you think that is already implied, and they can just click 'Save' to move along with the process?

Author: I don't want to overstep too much into the web builder process, especially because I think this doc is already overly arduous

> Return to :menuselection:`CRM app --> Sales --> Customers` and click the Kanban card for a partner.

Reviewer: ```suggestion
Next, return to :menuselection:`CRM app --> Sales --> Customers`, and click the Kanban card for a partner.
```
OPTIONAL: adding 'next' to keep the reader in the "flow" of the process being detailed//added a comma for enhanced sentence flow/pacing.

> Click the :guilabel:`Go to Website` smart button at the top of the page to open the partner's

Reviewer: ```suggestion
From that partner's contact form, click the :guilabel:`Go to Website` smart button at the top of the page to open that partner's
```

webpage.

> Click :guilabel:`Edit` at the top-right of the page, and use the :doc:`building blocks

Reviewer: ```suggestion
Next, click :guilabel:`Edit` at the top-right of the partner's webpage, and use the :doc:`building blocks
```

> <../websites/website/web_design/building_blocks>` to add any additional information about the

Reviewer: ```suggestion
<../websites/website/web_design/building_blocks>` to add any additional design elements, or information about the
```

partner.

.. tip::
  A company summary is a useful addition to this page.

After making any necessary changes to the page, click :guilabel:`Save`. At the top of the page,

> slide the `Unpublished` toggle to the active, `Published` position.

Reviewer 2: I think a gui label should be used here somewhere.

Repeat these steps for all partners.

Membership tab
--------------

> Finally, there is the :guilabel:`Membership` tab, which can help users manage any memberships that

Reviewer: ```suggestion
Finally, there is the :guilabel:`Membership` tab on contact forms, which can help users manage any memberships that
```
just for added specificity

> are being offered to this specific contact. It should be noted that this tab only appears when the

Reviewer: ```suggestion
are being offered to this specific contact. It should be noted that this tab **only** appears when the
```

> *Members* applications is installed.

Reviewer: ```suggestion
*Members* application is installed.
```

Activate membership
~~~~~~~~~~~~~~~~~~~

> To activate a contact's membership, click :guilabel:`Buy Membership`. On the pop-up window, select

Reviewer: ```suggestion
To activate a contact's membership, click :guilabel:`Buy Membership` in the :guilabel:`Membership` tab of a contact form. On the pop-up window that appears, select
```

> a :guilabel:`Membership` from the drop-down menu, then click :guilabel:`Invoice Membership`.

Reviewer: ```suggestion
a :guilabel:`Membership` from the drop-down menu. Then, configure a :guilabel:`Member Price`. Click :guilabel:`Invoice Membership` when both fields are filled in.
```

> Alternatively, to offer a free membership, tick the :guilabel:`Free Member` checkbox.

Reviewer: ```suggestion
Alternatively, to offer a free membership, tick the :guilabel:`Free Member` checkbox, in the :guilabel;`Membership` tab of a contact form.
```

> To publish a list of all active members on the website, a new webpage needs to first be created.

Reviewer 2: ```suggestion
To publish a list of all active members on the website, a new webpage must be created first.
```

> Navigate to :menuselection:`Website app --> Site --> Pages` and click :guilabel:`New`. Hover over

Reviewer: ```suggestion
To create a new webpage, navigate to :menuselection:`Website app --> Site --> Pages`, and click :guilabel:`New`. Hover over
```
You can also opt to mention that other route I mentioned above here, as well -- if you'd like...

the :guilabel:`Blank Page` template, and click the :guilabel:`Use this template` button to create a
new page.

> In the :guilabel:`New Page` pop-up window, enter `Members` in the :guilabel:`Page Title` field.

Reviewer: ```suggestion
In the :guilabel:`New Page` pop-up window, enter an appropriate title, like `Members`, in the :guilabel:`Page Title` field.
```

> Then click :guilabel:`Create`. In the :guilabel:`Edit` toolbar on the far right of the page, click

Reviewer 2: ```suggestion
Then click :guilabel:`Create`. In the :guilabel:`Edit` toolbar on the far-right of the page, click
```

> Then click :guilabel:`Create`. In the :guilabel:`Edit` toolbar on the far-right of the page, click

Reviewer: ```suggestion
Then, click :guilabel:`Create`. In the :guilabel:`Edit` toolbar on the far-right of the page, click
```

:guilabel:`Save`.

> Return to :menuselection:`CRM app --> Sales --> Customers` and click the Kanban card for a member.

Reviewer: ```suggestion
Return to :menuselection:`CRM app --> Sales --> Customers`, and click the Kanban card for a member.
```
SUGGESTION: added comma for enhanced sentence pacing/flow

> Click the :guilabel:`Go to Website` smart button at the top of the page to open the partner's

Reviewer: ```suggestion
Click the :guilabel:`Go to Website` smart button at the top of the page to open the member's
```

webpage.

After making any necessary changes to the page, click :guilabel:`Save`. At the top of the page,

> slide the `Unpublished` toggle to the active, `Published` position.

Reviewer 2: Same comment as above.
